docs/get-started/local-connection.md

```
@@ -19,11 +19,99 @@ If you are not sure if the MySQL server is running, open _Activity Viewer_ (from
 ```
 [mysqld]
 socket=/Users/YourUserName/Library/Containers/com.sequel-ace.sequel-ace/Data/mysql.sock
+
+[client]
+socket=/Users/YourUserName/Library/Containers/com.sequel-ace.sequel-ace/Data/mysql.sock
+```
+
+3. Restart MySQL/MariaDB.
+
+4. If startup fails with an error like `Could not create unix socket lock file`, verify permissions so the server process user can write in that directory:
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)